xlators/storage/posix/src/posix-helpers.c

```diff
@@ -135,8 +135,9 @@ posix_handle_georep_xattrs(call_frame_t *frame, const char *name, int *op_errno,
     int pid = 1;
     gf_boolean_t filter_xattr = _gf_true;
     static const char *georep_xattr[] = {
-        "*.glusterfs.*.stime", "*.glusterfs.*.xtime",
-        "*.glusterfs.*.entry_stime", "*.glusterfs.volume-mark.*", NULL};
+        "*.glusterfs.*.stime",       "*.glusterfs.*.xtime",
+        "*.glusterfs.*.lockinfo",    "*.glusterfs.*.entry_stime",
```

Comment:
Do we have anything in between `glusterfs.` and `.lockinfo`? all the other xattrs listed here will have some string after `glusterfs.` and actual name.

Reply:
No we will not be having a string. I suspected some changes in downstream code, so had added a * to accommodate if there were any.

Reply:
then I think match will fail to filter this xattr.

Reply:
Yes, that is right. The following patch set rectifies it.